plugins/board-resources/src/components/editor/CardDetails.svelte

```diff
@@ -74,7 +76,7 @@
         <Label label={board.string.Members} />
       </div>
       <div class="flex-row-center flex-gap-1">
-        {#each members as member}
+        {#each members as member (member._id)}
```

Comment:
```suggestion
        {#each members as member}
```
Please do not use if not desired, it will not update item if some other person will change member, like avatar etc.

Reply:
Done.